frontend/www/js/omegaup/arena/arena.ts

```diff
@@ -2377,7 +2426,7 @@ export class EventsSocket {
         socket.onmessage = (message) => this.onmessage(message);
         socket.onopen = () => {
           this.shouldRetry = true;
-          this.arena.elements.socketStatus.html('&bull;').css('color', '#080');
+          this.arena.updateSocketStatus('&bull;', '#080');
```

Comment:
```suggestion
          this.arena.updateSocketStatus('•', '#080');
```

para no tener que establecerlo mediante HTML.